src/utils/class-names.js

```diff
@@ -21,21 +21,39 @@ class ClassNames {
     return this
   }
 
-  withModule(module) {
+  from(module) {
     Object.assign(this.module, module)
     return this
   }
 
+  /**
+   * @deprecated
```

Comment:
Couldn't you remove it completely?

Reply:
I could but as a library developer I cannot be sure that this is not used anywhere, so I have to keep it.

Also, such approach allows me to migrate internal components later.

Reply:
I couldn't agree more but let's be honest here. We use this library only in @oacore/dashboard and we agreed it's a beta version. And everything may change. I propose to do this stuff once we release 1.0.0.

Reply:
I agree. On other hand, I have to modify all other component in all other PRs here to make it work. I do not want that. It worth to keep `@deprecated` for a few commits, start using new API and the migrate at some point all other components.